README.md

```diff
@@ -49,7 +58,74 @@ The `fairlearn` package contains the following algorithms for mitigating unfairn
+
+### Visualization: single model, pre-mitigation 
+
+Load the visualization dashboard in your noteboook to observe and assess your model's fairness using the following code:
+
+```python
+from fairlearn.widget import FairlearnDashboard
+
+# A_test containts your protected attributes (e.g., Age, Gender)
+# sensitive_feature_names containts your protected attributes names
+# y_true contains ground truth labels
+# y_pred contains prediction labels
+
+FairlearnDashboard(sensitive_features=A_test,
+                   sensitive_feature_names=['Gender', 'Age'],
+                   y_true=Y_test.tolist(),
+                   y_pred=[y_pred.tolist()])
+```
+
+
+Pre-mitigation visualization dashboard consists of procedural steps for configuration and evaluation of model fairness insights. In the configuration stage, you select your (1) protected (sensitive) attribute of interest (e.g., gender with three values of female, male, and non-binary), and (2) disparity metric that you would like to evaluate across different groups of your selected protected attribute (e.g., model precision). The evaluation stage uses these configurations to represent a detailed view of model's treatment across different subgroups of your selected protected attribute (e.g., model precision for females, model precision for males, model precision for people with non-binary gender). 
+
+
+
+The following plots provide a view of the configuration steps, where `gender` is selected as a protected attribute and `accuracy` is selected as the metric of interest to be analyzed for different subgroups of `gender`.
+
+
+
+![Configuration Steps](img/fairlearn-dashboard-config.png)
+
+
+
+
+
+|Plot|Description|
+|----|-----------|
+|Disparity in Accuracy| This panel (1) shows the accuracy of your model with respect to your selected accuracy metric (e.g., `accuracy`) (A) on the entire model, and (B) on different subgroups of your selected protected attribute (e.g., `accuracy` for females, `accuracy` for males, `accuracy` for datapoints with a non-binary gender), (2) highlights the disparity (difference) of selected accuracy metric across different subgroups of protected attribute. It also (3) shows the distribution of errors in each subgroup (e.g., female, male, non-binary gender). In the binary classification case, it will further split the errors into overprediction (predicting 1 when the true label is 0), and underprediction (predicting 0 when the true label is 1).
```

Comment:
'datapoints with a non-binary gender' -> probably should _really_ have 'individuals' rather than 'datapoints' here.